app/assets/javascripts/no_sunet.js

```diff
@@ -0,0 +1,43 @@
+$(document).on('click', '#no-sunet-button', function(){
+  setTimeout(function(){
+      $('#request_user_attributes_library_id').focus();
+  }, 1);
+
+  $('form').on('keypress', function(e) {
+    if(e.which == 13) {
+      $('.form-group input').checkForm(e);
```

Comment:
This simply references the first `input` in the first `.form-group`. Might we want to deal w/ non `input` form fields now or in the future?